src/elements/play-pen/play-pen.ts

```diff
@@ -444,7 +444,7 @@ export class PlayPen extends LitElement {
   }
 
   /** Throttled changes after updating sources. */
-  #setSrcSideEffects = throttle((save: boolean): void => {
+  #setSrcSideEffects = debounce((save: boolean): void => {
```

Comment:
I haven't tried this but trust your judgement :+1: the intent of the original design was to give regular feedback as fast as possible.

Reply:
The rollback would be easy in case we hate it, let's give it a try